descriptors/ModuleDescriptor-template.json

```diff
@@ -387,6 +387,8 @@
         "inventory-storage.material-types.collection.get",
         "inventory-storage.instances.item.get",
         "inventory-storage.instances.collection.get",
+        "inventory-storage.loan-types.collection.get",
```

Comment:
Do we declare dependency on interface `loan-types`?

Reply:
done